tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -50,7 +51,8 @@ sub run_test {
     my ($ACTIVE_POOL_NAME, $AVAILABLE_POOL_SIZE) = $self->get_active_pool_and_available_space();
     record_info('Detect Active POOL NAME:',    $ACTIVE_POOL_NAME);
     record_info('Detect Available POOL SIZE:', $AVAILABLE_POOL_SIZE . 'GiB');
-    assert_script_run("test $AVAILABLE_POOL_SIZE -ge 60", fail_message => "The SUT needs at least 60GiB available space of active pool for virtual network test");
+    my $expected_pool_size = (is_xen_host) ? '40' : '20';
```

Comment:
Perhaps it would be helpful to be able to set the expected pool size via a test SETTING, e.g.

    my $expected_pool_size = get_var('VIRT_EXPECTED_POOLSIZE', (is_xen_host) ? '40' : '20');

Reply:
Cool! I will updated this great idea.

Reply:
Had updated already.